### DISCUSSION TITLE
[Project Management] Process for onboarding new origins/providing support

### DISCUSSION THREAD

**Discussion by ChristinaLK** [PelicanPlatform/pelican#1140] [Ideas]
We need to formalize a (lightweight) onboarding process + "how to get support" document for initial partners in spinning up origins. 
Process needs to include getting / recording certain key information (organization, contact info, pelican status, etc.)

**Comment by turetske** [comment]
Right. I feel like the first step on our end should be creating a list of what is needed from the user to debug things.

The three that immediately come to mind are the configuration variables that are being used, the pelican version, and the actual command that's being run.
